File: tests/test-hello.cs
/*
 * This should give CORRECT on the default problem 'hello'. Note that
 * it will fail with RUN-ERROR if using the (default) chroot environment.
 *
 * @EXPECTED_RESULTS@: CORRECT
 */

using System;

public class Hello
{
	public static void Main(string[] args)
	{
#if ONLINE_JUDGE
		Console.Write("Hello world!\n");
#else
		Console.Write("ONLINE_JUDGE not defined\n");
#endif
	}
}

File: example_problems/hello/submissions/accepted/invalid-header/test-hello.c
/*
 * This should give CORRECT on the default problem 'hello'.
 * The extra, invalid, header-file will not be passed to gcc,
 * since it doesn't have a valid c extension. Thus, it will be
 * ignored during compilation, since it is not included in this file.
 *
 * @EXPECTED_RESULTS@: CORRECT
 */

#include <stdio.h>

int main()
{
	char hello[20] = "Hello world!";
	printf("%s\n",hello);
	return 0;
}
